public_suffix_list.dat

```diff
@@ -15504,6 +15504,11 @@ unison-services.cloud
 virtual-user.de
 virtualuser.de
 
+// United Tech : https://unitedtech.ai/
+// Submitted by Illia Korzh <illia.k@unitedtech.ai>
+*.pages-unitedtech.link
+*.dev-pages-unitedtech.link
```

Comment:
Please fix sorting, `dev-pages-unitedtech.link` should be above `pages-unitedtech.link`

Reply:
The wildcard character matches any valid sequence of characters in a hostname part. I assume you don't need that.